plugin/src/android/QonversionPlugin.java

```diff
@@ -89,15 +89,19 @@ public void purchase(
             @Nullable JSONArray contextKeys,
             CallbackContext callbackContext
     ) {
-        List<String> contextKeysList = EntitiesConverter.convertArrayToStringList(ids);
-        qonversionSandwich.purchase(
-            productId,
-            offerId,
-            applyOffer,
-            oldProductId,
-            updatePolicyKey,
-            contextKeysList,
-            Utils.getResultListener(callbackContext));
+        try {
+            List<String> contextKeysList = EntitiesConverter.convertArrayToStringList(contextKeys);
+            qonversionSandwich.purchase(
+                    productId,
+                    offerId,
+                    applyOffer,
+                    oldProductId,
+                    updatePolicyKey,
+                    contextKeysList,
+                    Utils.getResultListener(callbackContext));
+        } catch (JSONException e) {
+            e.printStackTrace();
```

Comment:
Was it added for debugging?

Reply:
No, it's the exception that can be thrown from `EntitiesConverter.convertArrayToStringList`.